### DISCUSSION TITLE
my request was denied: Unfortunately, we were unable to verify your academic affiliation based on the information yo

### DISCUSSION THREAD

**Discussion by kausthubh-coder** [community/community#52126] [GitHub Education]
I uploaded my student transcript which has a date and my name I also uploaded my student id, I also updated my profile to have my name, student email, and a bio none it worked.

![image](https://user-images.githubusercontent.com/69598472/230253906-53781eca-9208-4263-8c21-0a4f10313b77.png)

**Comment by samus-aran** [comment]
Hi @kausthubh-coder 👋 

Here are a few tips to help you in your application:

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know here if you need further assistance 👍
